engine/hid/src/dmsdk/hid/hid.h

```diff
@@ -0,0 +1,589 @@
+    * @member dmHID::KEY_A
+    * @member dmHID::KEY_B
+    * @member dmHID::KEY_C
+    * @member dmHID::KEY_D
+    * @member dmHID::KEY_E
+    * @member dmHID::KEY_F
+    * @member dmHID::KEY_G
+    * @member dmHID::KEY_H
+    * @member dmHID::KEY_I
+    * @member dmHID::KEY_J
+    * @member dmHID::KEY_K
+    * @member dmHID::KEY_L
+    * @member dmHID::KEY_M
+    * @member dmHID::KEY_N
+    * @member dmHID::KEY_O
+    * @member dmHID::KEY_P
+    * @member dmHID::KEY_Q
+    * @member dmHID::KEY_R
+    * @member dmHID::KEY_S
+    * @member dmHID::KEY_T
+    * @member dmHID::KEY_U
+    * @member dmHID::KEY_V
+    * @member dmHID::KEY_W
+    * @member dmHID::KEY_X
+    * @member dmHID::KEY_Y
+    * @member dmHID::KEY_Z
+    * @member dmHID::KEY_LBRACKET
+    * @member dmHID::KEY_BACKSLASH
+    * @member dmHID::KEY_RBRACKET
+    * @member dmHID::KEY_CARET
+    * @member dmHID::KEY_UNDERSCORE
+    * @member dmHID::KEY_BACKQUOTE
+    * @member dmHID::KEY_LBRACE
+    * @member dmHID::KEY_PIPE
+    * @member dmHID::KEY_RBRACE
+    * @member dmHID::KEY_TILDE
+    * @member dmHID::KEY_ESC
+    * @member dmHID::KEY_F1
+    * @member dmHID::KEY_F2
+    * @member dmHID::KEY_F3
+    * @member dmHID::KEY_F4
+    * @member dmHID::KEY_F5
+    * @member dmHID::KEY_F6
+    * @member dmHID::KEY_F7
+    * @member dmHID::KEY_F8
+    * @member dmHID::KEY_F9
+    * @member dmHID::KEY_F10
+    * @member dmHID::KEY_F11
+    * @member dmHID::KEY_F12
+    * @member dmHID::KEY_UP
+    * @member dmHID::KEY_DOWN
+    * @member dmHID::KEY_LEFT
+    * @member dmHID::KEY_RIGHT
+    * @member dmHID::KEY_LSHIFT
+    * @member dmHID::KEY_RSHIFT
+    * @member dmHID::KEY_LCTRL
+    * @member dmHID::KEY_RCTRL
+    * @member dmHID::KEY_LALT
+    * @member dmHID::KEY_RALT
+    * @member dmHID::KEY_TAB
+    * @member dmHID::KEY_ENTER
+    * @member dmHID::KEY_BACKSPACE
+    * @member dmHID::KEY_INSERT
+    * @member dmHID::KEY_DEL
+    * @member dmHID::KEY_PAGEUP
+    * @member dmHID::KEY_PAGEDOWN
+    * @member dmHID::KEY_HOME
+    * @member dmHID::KEY_END
+    * @member dmHID::KEY_KP_0
+    * @member dmHID::KEY_KP_1
+    * @member dmHID::KEY_KP_2
+    * @member dmHID::KEY_KP_3
+    * @member dmHID::KEY_KP_4
+    * @member dmHID::KEY_KP_5
+    * @member dmHID::KEY_KP_6
+    * @member dmHID::KEY_KP_7
+    * @member dmHID::KEY_KP_8
+    * @member dmHID::KEY_KP_9
+    * @member dmHID::KEY_KP_DIVIDE
+    * @member dmHID::KEY_KP_MULTIPLY
+    * @member dmHID::KEY_KP_SUBTRACT
+    * @member dmHID::KEY_KP_ADD
+    * @member dmHID::KEY_KP_DECIMAL
+    * @member dmHID::KEY_KP_EQUAL
+    * @member dmHID::KEY_KP_ENTER
+    * @member dmHID::KEY_KP_NUM_LOCK
+    * @member dmHID::KEY_CAPS_LOCK
+    * @member dmHID::KEY_SCROLL_LOCK
+    * @member dmHID::KEY_PAUSE
+    * @member dmHID::KEY_LSUPER
+    * @member dmHID::KEY_RSUPER
+    * @member dmHID::KEY_MENU
+    * @member dmHID::KEY_BACK
+    * @member dmHID::MAX_KEY_COUNT
+    */
+    enum Key
+    {
+        KEY_SPACE = HID_KEY_SPACE,
```

Comment:
These values are the reason I needed to add the glfw.h to the sdk.
(sdk headers shouldn't refer to internal headers)

These HID_* values are in turn defined as "HID_KEY_SPACE = GLFW_KEY_SPACE".
What we want in the future is to generate the mappings to the actual values, to cut the dependency: "HID_KEY_SPACE = 0xfafffaff" etc.